### DISCUSSION TITLE
纯好奇向基建问题

### DISCUSSION THREAD

**Discussion by johhnykk** [MaaAssistantArknights/MaaAssistantArknights#7714] [General]
如题，本人平常也不太去研究基建，但还是会好奇maa下基建的放置，之前有听说能天使、拉狗、德狗放在一起效率很高，但为什么maa会喜欢拆开来，是因为这样效率会更高吗。
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/147373245/b8e68b31-4f21-4e45-9086-8a1fe95c87fc)
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/147373245/fd666a4f-eb5a-4236-add8-d465afa418d4)

**Comment by zzyyyl** [comment]
能天使+德克萨斯+拉普兰德， 这是上个世纪的配置了吧（x

另外如果你没用基建作业，maa 是只考虑单设施最优的

**Reply by zzyyyl** [reply]
再另外，这三个放一起是加心情吧，效率好像没变化

**Reply by DLmaster361** [reply]
请注意你干员的专精情况，部分干员未专精可能会影响效率，导致MAA判断该组合应该拆开

**Reply by ABA2396** [reply]
> 请注意你干员的专精情况，部分干员未专精可能会影响效率，导致MAA判断该组合应该拆开

就算全满配这三个在一起效率也不是最高的，三人一起上班已经是很久之前的事情了

**Reply by DLmaster361** [reply]
确实，三位一起只有1.0出头的效率，其它搭配都已经达到1.2多了。不过这次maa给的这个搭配应该还是出于对专精的考虑，他目前的搭配只有0.9左右，更差

**Comment by wxhwen** [comment]
使用基建自动换班是不是要电脑一直挂着

**Reply by DLmaster361** [reply]
是的，MAA必须保持运行

**Reply by status102** [reply]
换班的时候开着就行了（不开着谁给你换班

**Comment by 18892bugjump** [comment]
是这样，不过这两套现在效率也不行就是了
